Sources/AST/Declaration/FunctionSignatureDeclaration.swift

```diff
@@ -42,6 +42,17 @@ public struct FunctionSignatureDeclaration: ASTNode {
     return modifiers.contains { $0.kind == kind }
   }
 
+  // MARK: - Equatable
+  public static func == (lhs: FunctionSignatureDeclaration, rhs: FunctionSignatureDeclaration) -> Bool {
+    return lhs.identifier.name == rhs.identifier.name &&
+      lhs.modifiers.map{ $0.kind } == rhs.modifiers.map{ $0.kind } &&
+      lhs.attributes.map{ $0.kind } == rhs.attributes.map{ $0.kind } &&
+      lhs.resultType?.rawType == rhs.resultType?.rawType &&
+      lhs.parameters.map({ $0.identifier.name }) == rhs.parameters.map({ $0.identifier.name }) &&
+      lhs.parameters.map({ $0.type.rawType }) == rhs.parameters.map({ $0.type.rawType }) &&
+      lhs.parameters.map({ $0.isInout }) == rhs.parameters.map({ $0.isInout })
+  }
```

Comment:
There are parentheses on some of these maps but not others. They should all be consistent.